lambda/lambdaUtils.mjs

```diff
@@ -47,14 +48,55 @@ export const create = async (args, abi, bytecode) => {
   return resultJson;
 };
 
+export const import_lambda = async (abi, name, address) => {
```

Comment:
```suggestion
export const importLambda = async (abi, name, address) => {
```

Reply:
done importLambda because import cant be done.